src/development/vortex/development/utils/data/scripts/visualization.py

```diff
@@ -0,0 +1,312 @@
+import vortex.development.utils.data.dataset.wrapper as vortex_wrapper
```

Comment:
I think this would be better place under main root script directory in [`scripts`](https://github.com/nodefluxio/vortex/tree/add-more-default-dataset/scripts) with a name `dataset_visualizer.py`

Reply:
ok